tests_v2/test_pane_break_swap_preserve_focus.py

```
@@ -36,31 +64,40 @@ def main() -> int:
+                lambda: client.current_workspace() == workspace_id,
+                "new workspace becomes the selected workspace",
+            )
 
             _ = client.new_split("right")
-            time.sleep(0.2)
-
-            panes_payload = client._call("pane.list", {"workspace_id": workspace_id}) or {}
-            panes = panes_payload.get("panes") or []
-            _must(len(panes) == 2, f"expected two panes after split: {panes_payload}")
+            panes = _wait_for(
+                lambda: (p := _panes(client, workspace_id)) if len(p) == 2 else None,
```

Ruff (0.15.17)

[error] Undefined name `p`

(F821)